src/components/tabs/Tabs.module.css

```
@@ -7,19 +7,49 @@
 	margin-inline-start: -1rem;
 	overflow-x: scroll;
 	/* Hide scrollbar on firefox */
-	scrollbar-width: none; 
+	scrollbar-width: none;
 }
 
 .tab-scroll-overflow::-webkit-scrollbar {
 	/* Hide scrollbar on chrome / safari */
-  display: none;
+	display: none;
 }
 
-.tab:focus {
+.tab,
+.tab:focus,
+.tab:active {
+	border-bottom: 0 !important;
+}
+
+.tab:focus-visible {
 	outline: 2px solid var(--theme-accent);
 	outline-offset: -2px;
 }
 
 .tabpanel {
 	padding-block-start: 1rem;
 }
+
+.tablist {
+	position: relative;
+}
+
+.selectedIndicator {
+	position: absolute;
+	bottom: -4px;
+	height: 4px;
+	background-color: var(--theme-accent);
+	transition-duration: 200ms;
+}
+
+@media (forced-colors: active) {
+	.selectedIndicator {
+		background-color: ButtonText;
+	}
+}
+
+@media (prefers-reduced-motion) {
+	.selectedIndicator {
+		transition-duration: 0;
+	}
+}
```

And here’s that change:

```suggestion
@media (prefers-reduced-motion: no-preference) {
	.selectedIndicator {
		transition-duration: 200ms;
	}
}
```